mobile-center-link-scripts/src/ios/MobileCenterConfig.js

```diff
@@ -40,7 +40,15 @@ function addConfigToProject(file) {
     return new Promise(function (resolve, reject) {
         debug(`Trying to add ${file} to XCode project`);
 
-        var globString = `**/${(pjson && pjson.name ? pjson.name : '*')}.xcodeproj/project.pbxproj`
+        var globString;
+        if (pjson && pjson.name) {
+            globString = `**/${(pjson.name.replace(/-/g, ''))}`;
+        }
+        else {
+            globString = `ios/*`;
+        }
+        globString += '.xcodeproj/project.pbxproj';
+
         var projectPaths = glob.sync(globString, { ignore: 'node_modules/**' });
```

Comment:
Hi @guperrot, in general looks good to me. Several questions:
1) If the only reason is to get rid of dashes in name does it make sense to use the following short form:
 `var globString = **/${(pjson && pjson.name ? (pjson.name.replace(/-/g, '')) : '*')}.xcodeproj/project.pbxproj`?
Also you have replaced search pattern for `ios/*.xcodeproj` instead of usage of globstar `**/*.xcodeproj`.  
In fact for most cases it should be indeed in `ios` folder but theoretically it can be in other place. What do you think?
2) Just my interest . When tried to create app name with dashes in name using `react-native init` or `rninit init` commands but it says that `it is not valid name for a project. Please use a valid identifier name (alphanumeric). ` So the case when using dashes in names is when the app was renamed intentionally or react was added to existing app, am I correct?